src/swarm-game/Swarm/Game/Util/Failure.hs

```diff
@@ -11,7 +11,7 @@
 --
 -- These failures are often not fatal and serve
 -- to create common infrastructure for logging.
-module Swarm.TUI.Model.Failure where
+module Swarm.Game.Util.Failure where
```

Comment:
@kostmo FYI I had to move the module back to the Game just to compile, but the change in #1116 is better (no `Util` directory and less code in the `Util` module itself) so I will rebase once merged. :+1: 

It seems that I am also susceptible to making silly `Util` modules when desperate. :sweat_smile: 

Splitting the libraries with Cabal has been a great learning experience for me, so I hope not to repeat my mistake from #982 where I put this module in TUI in the first place. :facepalm: